packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0-nullsafety
+
+* Migrated to null-safety.
```

Comment:
nits: Let's also mention the "BREAKING CHANGE", which is removed deprecated APIs.

Reply:
Done.